website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -35,6 +43,49 @@ When multiple policies/groups/users are attached, you can configure the _Policy
 - Require users to pass all bindings/be member of all groups (ALL), or
 - Require users to pass either binding/be member of either group (ANY)
 
+## Application Entitlements
+
+<span class="badge badge--preview">Preview</span>
+<span class="badge badge--version">authentik 2024.12+</span>
+
+Application entitlements can be used through authentik to manage authorization within an application (what areas of the app can users or groups can access). Entitlements are scoped to a single application and can be bound to multiple users/groups (binding policies is not currently supported), giving them access to the entitlement. An application can either check for the name of the entitlement (via the `entitlements` scope), or via attributes stored in entitlements.
+
+An authentik admin can create an entitlement [in the Admin interface](#create-an-application-entitlement) or using the [authentik API](../../developer-docs/api/api.md).
+
+Because entitlements exist within an application, names of entitlements must be unique within an application. This also means that entitlements are deleted when an application is deleted.
```

Comment:
Suggestion: Wordy.  "Entitlements associated with an application will be deleted when the application is deleted."
Alternatively, "Entitlements bound to an application..."